src/add_forward_poly_association_plugin.ts

```diff
@@ -37,18 +39,21 @@ export const addForwardPolyAssociation = (builder: SchemaBuilder, option: Option
     // Find  all the forward relations with polymorphic
     const forwardPolyRelationSpec = (
       <PgPolymorphicConstraints>pgPolymorphicClassAndTargetModels)
-      .filter(con => con.from === table.id)
+      .filter(con => canonical(con.from) === canonical(table.id))
       .reduce((memo, currentPoly) => {
         const { name } = currentPoly;
         const sourceTableId = `${name}_id`;
         const sourceTableType = `${name}_type`;
         const fieldsPerPolymorphicConstraint = currentPoly.to.reduce((acc, mName) => {
-          const pgTableSimple = mapFieldToPgTable[mName];
+          const tableName:string = Object.keys(mapFieldToPgTable).find(n => canonical(n) == canonical(mName)) || mName;
```

Comment:
This may not be necessary, because the mapFieldToPgTable is constructed only to map the graphql model name back to the pg table name. Unless you also have inflections that create these canonical graphql type names, you shouldn't need that ?

Reply:
I think this had to do with the fact that _type columns refer to the Rails-y (i.e. module-namespaced) names.